### ISSUE TITLE
br-wan has mesh IPv6 addresses; fastd over IPv6 stops working after some time

### ISSUE THREAD

**Issue by ohrensessel** [freifunk-gluon/gluon#189]
I'm trying out fastd over IPv6. On one of our gateways there is a 2nd fastd instance (gw05v6.hamburg.freifunk.net) on a different port (10001) and with an adapted MTU (1406).

However, connecting from a node only works for about 10min, afterwards the handshakes are not answered and fastd switches to IPv4 again. The node is running v2014.3-67-gbb6d0f1.
As per `ip a` br-wan not only has an IPv6 address from my local network but also from the mesh network, which shouldn't be the case.

`logread | grep fastd` http://pastebin.com/uAqdS4qV
`logread` http://pastebin.com/TuHaYa0V
`cat /etc/config/network` http://pastebin.com/7mxCvzvK
`ubus call network.interface dump` http://pastebin.com/W2YFbitn
`ip -6 r s t 1` http://pastebin.com/CpFVcQPm
`ip a` http://pastebin.com/H9hiW5AY

**Comment by ohrensessel** [comment]
having this issue again.

- RAs in my local network: only from my fritzbox, only containing the ipv6 prefix assigned by my provider --> everything correct here
- Manually removing the freifunk internal prefixes from br-wan works, but the prefixes reappear after some time
- having issues with KD at the moment, loosing internet connectivity every 10-20mins. somehow the freifunk prefixes reappearing on br-wan is connected to the periods of no internet connectivity.
- logread contains nothing but "fastd connection ... disestablished" when internet connectivity is lost and entries by dnsmasq "dnsmasq[1273]: reading /var/gluon/wan-dnsmasq/resolv.conf"

**Comment by Barbarossa-** [comment]
We've seen some issues with FritzBox!es where they somehow saw the FF-net on the network and reannounced it into the LAN. That might be an explanation for this problem as the FF node might then get an RA from the FritzBox for the FF-network and configure the address accordingly.

(Barbarossa, FFPB)

**Comment by Barbarossa-** [comment]
Just saw RAs originating from a FritzBox announcing our FF network as "onlink" besides the correct regular network. We'll try to reprocude that situation.

**Comment by oscarminus** [comment]
I've had the same issue. My notebook was connected to the freifunk network. Then i changed the wireless network to my private network. The ipaddress still resists on wlan0 at that point of time. A day after, i noticed that my notebook had an ipadress of our freifunk network on wlan0, marked as deprecated, although i was connected to my private wlan. I couldn't explain that to me and started tcpdump. The dump showed, that the according RAs came from my FritzBox. I was wondering how this could happen. After a reboot of the FritzBox, the ffpb-RAs disappeared. Connecting the Notebook to Freifunk and back to my private network, everything starts again. 

After some time, i figured out, that my fritzbox has an option called "Auch IPv6-Präfixe zulassen, die andere IPv6-Router im Heimnetzwerk bekanntgeben". Activating this option solves this problem with the deprecated RAs in my private network. It seems that otherwise the FritzBox tries to suppress foreign prefixes in "her" wlan. 

tl;dr; It seems that the issue, freifunk prefix on the wan port, is an issue of a FritzBox on the wan-site, not of the freifunk node.

**Comment by NeoRaider** [comment]
@oscarminus, can you try to find out how the FritzBox even gets the Freifunk prefix? Is it sufficient to have a client with a Freifunk address switch to the private LAN and generate traffic, or is there also something wrong with the Freifunk node which causes it to use the Freifunk prefix on the WAN port in such a situation? Getting some packet dumps in the private network on the client and the node's WAN port would probably be helpful.

**Comment by NeoRaider** [comment]
Closing, please reopen when you have new information on this.
